extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/PolicyMappingConfig.java

```diff
@@ -60,4 +60,13 @@ public class PolicyMappingConfig {
      */
     @ConfigItem
     public Optional<String> authMechanism;
+
+    /**
+     * Shared permission checks are always applied whenever the path is matched. On the other hand, there is always
```

Comment:
@michalvavrik I suggest to simplify a bit here and expand if necessary in the docs, as it reads a little bit complex. How about something like `Indicates that this policy always applies to the matched paths in addition to the policy with a winning path. Avoid creating more than one shared policy to minimize the performance impact.` ?

Reply:
Sure, makes sense, fixed.